tests/fixtures/parsers/typescript-parsers/new-parens.js

```diff
@@ -0,0 +1,262 @@
+"use strict";
+
+// new Storage<RootState>('state');
+
+exports.parse = () => ({
+    "type": "Program",
```

Comment:
Will do. If you want to enforce this, should we just unignore the fixtures or so?